### PR TITLE
Twilio Flex support

### PR DESCRIPTION
**Pull request by cubeghost**
in combination w/ [plugin-bedstuystrong-ticket](https://github.com/bedstuystrong/plugin-bedstuystrong-ticket)

### CODE REVIEW

File: functions/airtable.js [hunk 1]
```diff
@@ -127,6 +127,14 @@ async function getRecordsWithStatus(table, status) {
   return await getRecordsWithFilter(table, { status });
 }
 
+async function getRecordsWithPhoneNumber(table, phoneNumber) {
+  const query = base(table).select({
+    filterByFormula: `{Phone Number} = "${phoneNumber}"`
+  });
+  const records = await query.all();
+  return records.map(normalizeRecords(table));
```
Comment (leifwalsh) [inline functions/airtable.js:135]:
```suggestion
  return await getRecordsWithFilter(table, { phoneNumber });
```

File: public/twilio/index.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+      display: none;
+      padding: 1rem;
+    }
+    #loading.visible {
+      display: block;
+    }
+  </style>
+</head>
+<body>
+  <template id="template">
```
Comment (leifwalsh) [inline public/twilio/index.html:87]: Never seen someone template html like that before, but cool.

**Review by leifwalsh — APPROVED** [review]
I'd take the suggestion but you can merge like this if you're lazy.

File: functions/airtable.js (hunk 1 above)
Comment (cubeghost) [reply functions/airtable.js]: oh nice!

**Comment by cubeghost** [comment]
removed the `/twilio` page in favor of having the flex plugin call the api directly: 
https://github.com/bedstuystrong/plugin-bedstuystrong-ticket/blob/master/src/states/index.js#L15-L27
https://github.com/bedstuystrong/plugin-bedstuystrong-ticket/blob/master/src/components/Ticket.js

**Pull request merged**